cmd/opampsupervisor/e2e_test.go

```diff
@@ -502,12 +498,6 @@ func TestSupervisorStartsCollectorWithRemoteConfigAndExecParams(t *testing.T) {
 
 	waitForSupervisorConnection(server.supervisorConnected, true)
 
-	// verify that the supervisor reports the remote config as applied and the correct hash
```

Comment:
Just so I fully understand, why remove this check from this test? Is it because we're testing it elsewhere?